index-scheduler/src/batch.rs

```diff
@@ -689,7 +696,67 @@ impl IndexScheduler {
 
                 Ok(tasks)
             }
+            Batch::IndexSwap { mut task } => {
+                let mut wtxn = self.env.write_txn()?;
+                let swaps = if let KindWithContent::IndexSwap { swaps } = &task.kind {
+                    swaps
+                } else {
+                    unreachable!()
+                };
+                for (lhs, rhs) in swaps {
+                    self.apply_index_swap(&mut wtxn, task.uid, lhs, rhs)?;
+                }
+                wtxn.commit()?;
+                task.status = Status::Succeeded;
```

Comment:
Should we have details here? I thought that we must have the original swap list.